Services/UserService.cs

```diff
@@ -0,0 +1,44 @@
+using TravelTrack_API.Models;
+
+namespace TravelTrack_API.Services;
+
+public static class UserService
```

Comment:
You should take static off of these classes. You'll want to use .NET's built in dependency injection to use the services in your controllers. You'll set that up in your program.cs file. Static is bad for testing and really doesn't gain us anything here.

Reply:
static references removed, dependency injection implemented (not yet with DTOs, context, mapper, etc.)